pom.xml

```diff
@@ -16,16 +16,11 @@
   <name>guacamole: variant caller</name>
 
   <properties>
-    <adam.version>0.19.0</adam.version>
```

Comment:
Yea, I don't think any of these qualify as "major dependencies".

The two that do, spark and hadoop, coincidentally (or not!) are used in multiple places.